.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,15 @@
 <!--
-Thanks for contributing a pull request! Please ensure you have taken a look at
-the contribution guidelines: https://github.com/pydicom/pydicom/blob/master/CONTRIBUTING.md#contributing-pull-requests
+Please prefix your PR title with [WIP] for PRs that are in progress and change
+to [MRG] when you consider them ready for final review.
 -->
-#### Reference Issue
-<!-- Example: Fixes #1234 -->
 
+#### Describe the changes
+The related issue or a description of the bug or feature that this PR addresses.
 
-#### What does this implement/fix? Explain your changes.
-<!--
-Please summarize the key points of the reference issue, problem or contribution
-to facilitate reviewing task. Of course reviewers can always refer to the
-original issue but facilitating the reviewing process is much appreciated.
--->
-
-#### Any other comments?
-<!--
--->
-
-<!--
-Please be aware that we are a loose team of volunteers so patience is
-necessary; assistance handling other issues is very welcome. We value
-all user contributions, no matter how minor they are. If we are slow to
-review, either the pull request needs some benchmarking, tinkering,
-convincing, etc. or more likely the reviewers are simply busy. In either
-case, we ask for your understanding during the review process.
-
-Thanks for contributing!
--->
+#### Tasks
+- [ ] Unit tests added that reproduce the issue or prove feature is working
+- [ ] Fix or feature added
+- [ ] Documentation updated (if relevant)
+  - [ ] No warnings during build
+  - [ ] Preview link (CircleCI -> Artifacts -> `[...]/_build/html/index.html`)
+- [ ] Unit tests passing and PR coverage above 90% after adding fix/feature
```

Comment:
Is this value OK? Contributing guide says 80% but we're already at 88% and I'd prefer to aim at 100%

Reply:
Hm, aiming at 100% seems a bit over eager to me, but that's my personal feeling. I would probably formulate this less strict (something like "aim at a coverage of..."), though I would @darcymason let decide this.

Reply:
Perhaps just that coverage should not decrease? That has been my working expectation.  One day we can strive for 100% but that seems like it can come from a concerted effort.

Reply:
Yes, agreed. This is also what codecov checks (well, would check if it would work reliably...)

Reply:
OK, I'll change it to "overall coverage the same or better"